conf/modules.config

```diff
@@ -724,46 +723,16 @@ process{
         ]
     }
 
-    withName: 'MPILEUP_.*' {
-        ext.when         = { params.tools && params.tools.contains('controlfreec') }
+    withName: 'SAMTOOLS_MPILEUP' {
+        ext.when         = { params.tools && (params.tools.contains('controlfreec') || params.tools.contains('mpileup')) }
         publishDir       = [
             mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/${meta.id}/controlfreec" },
+            path: { "${params.outdir}/variant_calling/${meta.id}/mpileup" },
             pattern: "*mpileup",
             saveAs: { meta.num_intervals > 1 ? null : it }
         ]
     }
 
-    //CNVKIT
-    withName: 'CNVKIT_ANTITARGET' {
-        ext.when         = { params.tools && params.tools.contains('cnvkit') }
-        publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/reference" },
-            pattern: "*{bed}"
-        ]
-    }
-
-    withName: 'CNVKIT_REFERENCE' {
-        ext.prefix       = "cnvkit"
-        ext.when         = { params.tools && params.tools.contains('cnvkit') }
-        publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/reference" },
-            pattern: "*{cnn}"
-        ]
-    }
-
-    withName: 'CNVKIT_BATCH_TUMORONLY' {
-        ext.args         = { params.wes ? "--method hybrid --diagram --scatter" : "--method wgs --diagram --scatter" }
-        ext.when         = { params.tools && params.tools.contains('cnvkit') }
-        publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/${meta.id}/cnvkit" },
-            pattern: "*{bed,cnn,cnr,cns,pdf,png}"
-        ]
-    }
```

Comment:
What's happening with CNVKIT?

Reply:
I forgot to pull once again...fixing that right now